src/openforms/js/components/form/map.js

```diff
@@ -62,6 +62,80 @@ const EDIT_FORM_TABS = {
         HIDDEN,
         CLEAR_ON_HIDE,
         {...IS_SENSITIVE_DATA, defaultValue: true},
+        {
+          type: 'number',
+          key: 'defaultZoom',
+          label: 'Default zoom level.',
+          description: 'Default zoom level for the leaflet map.',
+          validate: {
+            min: 1,
+            max: 18,
+          },
+        },
+        {
+          type: 'columns',
+          input: false,
+          label: 'starting location',
+          key: 'startPop',
+          tooltip: 'The Latitude and Longitude of the starting point of the map component.',
+          defaultValue: [{lat: '', lng: ''}],
+          columns: [
```

Comment:
I don't particularly like columns due to the dependence on bootstrap - can we instead use a panel like `PRESENTATION` in `js/components/form/edit/options.js`?